### ISSUE TITLE
Tips for using for the task of Action detection

### ISSUE THREAD

**Issue by aelnouby** [NVIDIA/flownet2-pytorch#17]
I want to use flowNet 2.0 for training a two stream model for action detection (spatio-temporal localization), I want to achieve real time performance, I have 2 questions:

1. Which network do you recommend using, FlowNetS or FlowNetSD, I assume SD should be better since it was designed to help get better result on realistic datasets as UCF101 which I will be using for my task.

2. If I want to fine tune the network for my task, what learning rate do you think would be reasonable (I was thinking of around 1e-7) ?

**Comment by fitsumreda** [comment]
Hi @aelnouby ,

I don't think any of the models are real-time. If you are interested more in speed vs accuracy. I'd suggest FlowNetC, which gets you the advantage of picking large displacement. In the event that the motion content is small for your purpose, then FlowNet-SD would give you the advantage of picking small displacement. 

For a fine-tuning purpose, a learning rate <1e-6 would make more sense. Your suggestion sounds good.

**Comment by aelnouby** [comment]
Thanks for your reply.

I thought FlowNet-S processes 140 fps on a GPU as reported in the paper, so shouldn't this be good for a real time application ?

**Comment by fitsumreda** [comment]
I think the 140fps is for FlowNet-s (with a small letter). The FlowNet-S (with a capital later, which this repo implemeted) speed is ~56fps. 
The topology of the networks is the same, the difference is that FlowNet-S's channels are shrinked by a factor of 3/8 to make them FlowNet-s. Below is a table from the FlowNet-2 paper.

![image](https://user-images.githubusercontent.com/15355345/34450634-0d749360-ecc3-11e7-8e8d-ccce44468e27.png)

**Comment by aelnouby** [comment]
I see, I have a better understanding now. Thanks a lot for your very informative answer.
